app/graphql/server.js

```diff
@@ -5,15 +5,15 @@ import { createSchema } from './schema.js'
 
 export const schema = await createSchema()
 
+export const enableApolloLandingPage = () => {
+  if (process.env.GRAPHQL_DASHBOARD_ENABLED === 'true') {
```

Comment:
```suggestion
export const enableApolloLandingPage = (enabled) => {
  if (enabled === 'true') {
```
If you replace the env var with a param it might make the testing slightly nicer :shrug: